deps/hdr_histogram/hdr_alloc.h

```diff
@@ -0,0 +1,50 @@
+
+/**
+ * hdr_alloc.h
+ * Written by Filipe Oliveira and released to the public domain,
+ * as explained at http://creativecommons.org/publicdomain/zero/1.0/
+ *
+ * Allocator selection.
+ *
+ * This file is used in order to change the HdrHistogram allocator at run
+ * time. */
+
+#ifndef HDR_ALLOC_H
+#define HDR_ALLOC_H
+
+#include <stddef.h> /* for size_t */
+#include <stdint.h>
+
+/* Structure pointing to our actually configured allocators */
+typedef struct hdrAllocFuncs {
+  void *(*mallocFn)(size_t);
+  void *(*callocFn)(size_t, size_t);
+  void *(*reallocFn)(void *, size_t);
+  void (*freeFn)(void *);
+} hdrAllocFuncs;
```

Comment:
i see hdr_histogram uses indentation of 4 spaces (like redis), let's use it's style.

Reply:
:+1: addressed in the new commit
To achieve this I've used clang-format with the following defs:
```
IndentWidth: 4
AccessModifierOffset: -4
```